src/shogun/classifier/svm/LibLinear.cpp

```diff
@@ -52,15 +52,16 @@ void CLibLinear::init()
 
 	SG_ADD(&C1, "C1", "C Cost constant 1.", ParameterProperties::HYPER);
 	SG_ADD(&C2, "C2", "C Cost constant 2.", ParameterProperties::HYPER);
-	SG_ADD(
-	    &use_bias, "use_bias", "Indicates if bias is used.");
+	SG_ADD(&use_bias, "use_bias", "Indicates if bias is used.");
 	SG_ADD(&epsilon, "epsilon", "Convergence precision.");
-	SG_ADD(
-	    &max_iterations, "max_iterations", "Max number of iterations.");
+	SG_ADD(&max_iterations, "max_iterations", "Max number of iterations.");
 	SG_ADD(&m_linear_term, "linear_term", "Linear Term");
 	SG_ADD(
```

Comment:
This would be removed or?

Reply:
You mean it would be removed if we merge SG_ADD and SG_ADD_OPTIONS?

Reply:
Yes, to further remove redundant code